_config/site.yml

```diff
@@ -68,10 +68,11 @@ profiles:
 
   staging: 
     <<: *profile
-    dcp_base_protocol_relative_url: //dcpbeta-searchisko.rhcloud.com/
-    dcp_base_url: http://dcpbeta-searchisko.rhcloud.com/
-    dcp2_base_url: http://dcp2-searchisko.rhcloud.com/
-    dcp2_base_protocol_relative_url: //dcp2-searchisko.rhcloud.com/
+    dcp_base_protocol_relative_url: //dcp.jboss.org/
+    dcp_base_url: http://dcp.jboss.org/
+    dcp2_base_protocol_relative_url: //dcp2.jboss.org/
+    dcp2_base_url: http://dcp2.jboss.org/
+    push_to_searchisko: false
```

Comment:
What's the reasoning here? I see you are now pointing the staging build at the production DCPs. Why did you need to do that? I see why disabling push_to_searchisko is right in this case, as we don't want the staging build pushing to the production DCP.